index.html

```diff
@@ -0,0 +1,215 @@
+                 </p>
+            </div>
+            <footer>
+               <ul id="skills_1" class="skills">
+                 <li>html</li>
+                 <li>css</li>
+                 <li>javaScript</li>
+               </ul>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section id="p3">
+            <div id="introd_3">
+                  <img  class="project_image" src="./images/Snapshoot Portfolio (2).svg" alt="snapshoot image project">
+                  <h1>Tonic</h1>
+                  <div id="general_inf_2">
+                      <p class="title">CANOPY</p>
+                      <img class="counter" src="./images/Counter (1).svg" alt="counter"/>
+                      <p class="job-title">Back End Dev</p>
+                      <img class="counter" src="./images/Counter (1).svg" alt="counter">
+                      <p class="year">2015</p>
+                  </div>
+              
+                 <p class="project_description">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                 </p>
+            </div>
+            <footer>
+               <ul id="skills_2" class="skills">
+                 <li>html</li>
+                 <li>css</li>
+                 <li>javaScript</li>
+               </ul>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section  id="p4">
+            <div id="introd_4">
+                  <img  class="project_image" src="./images/Snapshoot Portfolio (3).svg" alt="snapshoot image project">
+                  <h1>Multi-Post
+                    Stories</h1>
+                  <div id="general_inf_3">
+                      <p class="title">CANOPY</p>
+                      <img class="counter" src="./images/Counter (1).svg" alt="counter"/>
+                      <p class="job-title">Back End Dev</p>
+                      <img class="counter" src="./images/Counter (1).svg" alt="counter">
+                      <p class="year">2015</p>
+                  </div>
+              
+                 <p class="project_description">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                 </p>
+            </div>
+            <footer>
+               <ul id="skills_3" class="skills">
+                 <li>html</li>
+                 <li>css</li>
+                 <li>javaScript</li>
+               </ul>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          </section>
+          <section id="about_me">
```

Comment:
- [ ] [REQUIRED] Good job so far! I really appreciate what you did 👍, but you need to add some `space` on both sides of the `Get my resume` button in the `About-myself` section, like the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), because the `text` is touching the `border lines` of the button. There is also the same error on the `right side` of the elements of the `Languages` category, they are touching the border lines and the location of the `arrow` icons should be parallel, just like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), but their `location` is different and their `sizes` are different. Please fix these problems 🙏🙏🙏

- ### Your design ❌ 

![image](https://user-images.githubusercontent.com/63915024/199779933-45de74b7-1fbd-4bdf-8b82-45db9c2a9d04.png)

- ### Figma design ✔️   

![image](https://user-images.githubusercontent.com/63915024/199779961-fafe4ee1-55b6-4c5b-969e-c1adf13e2232.png)